templates/repo/issue/labels/labels_selector_field.tmpl

```diff
@@ -0,0 +1,49 @@
+<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-label dropdown">
+	<a class="text gt-df gt-ac muted">
+		<strong>{{.locale.Tr "repo.issues.new.labels"}}</strong>
+		{{if and .HasIssuesOrPullsWritePermission (not .Repository.IsArchived)}}
+			{{svg "octicon-gear" 16 "gt-ml-2"}}
+		{{end}}
+	</a>
+	{{if .Issue}}
+	<div class="filter menu" data-action="update" data-issue-id="{{$.Issue.ID}}" data-update-url="{{$.RepoLink}}/issues/labels">
+	{{else}}
+	<div class="filter menu" data-id="#label_ids">
+	{{end}}
```

Comment:
Could it be something like `<div class="filter menu" {{if .Issue}}...{{else}}...{{end}}>`?

It's more friendly to editors, the `div` tags could be matched by IDE's syntax analyzer. Otherwise, IDE would see `<div><div></div>`